api/core/v2/event.proto

```diff
@@ -27,6 +30,9 @@ message Event {
   // Check describes the result of a check; if event is associated to check execution.
   Check check = 3 [(gogoproto.nullable) = true];
 
-  // Metrics ...
+  // Metrics are zero or more Sensu metrics
   Metrics metrics = 4 [(gogoproto.nullable) = true];
+
+  // Metadata contains name, namespace, labels and annotations
+  ObjectMeta metadata = 5 [(gogoproto.embed) = true, (gogoproto.jsontag) = "metadata", (gogoproto.nullable) = false];
```

Comment:
Metadata is likely never empty, but I noticed the other resources you refactored have `omitempty`. Is this something we want for events as well?

Reply:
It turns out that `omitempty` has no effect for structs.